libraries/AP_Compass/AP_Compass.h

```diff
@@ -204,8 +204,11 @@ friend class AP_Compass_Backend;
     float get_declination() const;
 
     // set overall board orientation
-    void set_board_orientation(enum Rotation orientation) {
+    void set_board_orientation(enum Rotation orientation, Matrix3f* custom_rotation = nullptr) {
         _board_orientation = orientation;
+        if (custom_rotation) {
```

Comment:
this should be unconditional, so if the user changes away from a custom rotation to a non-custom the variable gets set back to nullptr

Reply:
done, good point